docs/platforms/apple/guides/ios/size-analysis/index.mdx

```diff
@@ -12,7 +12,7 @@ description: Upload iOS builds to Sentry for size analysis.
 
 **Accepted Formats**: XCArchive (preferred) | IPA
 
-**Upload Mechanisms**: [Fastlane Plugin](#uploading-with-fastlane) (version 1.34.0 or higher) _or_ [Sentry CLI](#uploading-with-the-sentry-cli)
+**Upload Mechanisms**: [Fastlane Plugin](#uploading-with-fastlane) _or_ [Sentry CLI](#uploading-with-the-sentry-cli)
```

Comment:
`m`: Why did you remove the minimum version of the fastlane plugin?

Reply:
Good question, its redundant since we also mention the number in the `size-analysis/upload-fastlane` included below. 

<img width="634" height="337" alt="Image" src="https://github.com/user-attachments/assets/eca298a3-9b64-4696-9d11-016bab84a3e0" />